docker/benchmark-builder/Dockerfile

```diff
@@ -45,6 +45,12 @@ ENV ASAN_OPTIONS="detect_leaks=0"
 RUN mkdir /opt/fuzzbench/
 COPY docker/benchmark-builder/checkout_commit.py /opt/fuzzbench/
 COPY benchmarks/$benchmark/benchmark.yaml /
+# Use multiple copy on benchmark.yaml and the testcases folder, so as to not throw an error in case the folder doesn't exist
```

Comment:
If we check that testcases exists in generate_makefile, I'm not sure why this clever hack is needed.

Reply:
Because the benchmark-builder is run on all the benchmarks, even those without a testcases folder. To avoid generating a whole different image just to check reproducibility, I modified the normal builder to copy the folder if present, and then the folder will be used when the repro-check make rule is called.

Reply:
But please, do tell me if there is a better way to copy the testcases folder into the final image, because I don't like this hack either, I just didn't find a way to avoid using it.

Reply:
Yes there is a better way. 
Please get rid of these changes entirely.
Then when you start the container add something like ` -v /testcases:$PATH_TO_TESTCASES_ON_HOST` this tells docker to mount `$PATH_TO_TESTCASES_ON_HOST` to `testcases` as a volume, so the testcases can be accessed.